Description: fix compile warning: implicit declaration of function ‘strcmp’
 [-Wimplicit-function-declaration]
 .

---

Origin: xiao sheng wen <atzlinux@sina.com>
Last-Update: 2020-05-05

--- xtail-2.1.orig/xtail.c
+++ xtail-2.1/xtail.c
@@ -13,6 +13,7 @@
 #endif
 #define  INTERN
 #include "xtail.h"
+#include <string.h>
 
 int sigcaught = 0;
 RETSIGTYPE sigcatcher(int sig);
